### DISCUSSION TITLE
Education Ideas about DCDAO

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#23] [Ideas]
# Introduction

Since the establishment of DCDAO, I have had a different understanding of dcdao's vision, so let me share some thoughts on DCDAO below. For me, community means some people are gathered for some reason while DAO is some people who have a common vision and come together to create something. Based on this explanation, DCDAO can be divided into four parts as the following:

1. Co-learning 
2. Co-working 
3. Co-creating 
4. Others

I will explain each part step by step.  Share your ideas in the comments.🙂

# 1. Co-learning

## 1.1 Why

Co-learning aims at making the understanding of the common vision more clear and deeper. 

We believe **“*When the student is ready the teacher will appear*”** and *“**A positive and effective relationship between a teacher and student should naturally flow.”***

We hope the way to learn is not only to seek a good university or a decent job but also to become an **initiative, interactive and interesting** process. 

So DCDAO School is created. 

## 1.2 What

In school, there are three key points to learn, that is “Web3”, “Darwinia”, and “DCDAO”.

You can find the study resources as your wish in this school library and everyone can be a teacher or student.   

## 1.3 How

In terms of how to build this school, it includes two parts as the following:

1. school curriculum
2. education system

Based on DCDAO School’s beliefs and education vision, we hope these can be created by all the members and ambassadors.

As mentioned, a sound relationship between the teacher and student is flow. The knowledge distribution should be not only replied to top-down but also the bellow. 

So the school's curriculum can be fragmented which is based on individual interests. Or it can be systematic, generated by the organization. Like this famous picture.
![800](https://github.com/dcdao/discussion/assets/109792098/9c7b0f9b-0e5c-45e2-8cf8-51b1da88a8e6)
# 2. Co-working (Continue to Be Updated)

## 2.1 Town Hall

Town hall mainly solves public problems for members. By now, the problems contain community questions, media affairs, etc. 

## 2.2 Others

eg: Community RPC Experimental Program

# 3. Co-creating(Incubator Program)

## 3.1 Why

Make community resources flow.

## 3.2 What

DCDAO's incubator program provides resources and support for startups working on blockchain technology and other decentralized solutions.

## 3.3 How

eg: Bounty Board

Bounty Board: A platform within the DCDAO where members can create or select the task to complete, often in exchange for RING.

# 4. Others

Stay tune~

**Comment by SasoLithops** [comment]
Hi!

I love the tech Darwinia offers. But there are many things too technical for me to understand. So I would suggest if we could have a Channel where we can talk directly to team leaders and devs to learn more. It was Megan for me who taught me about tech in the way I could understand. And since Darwinia is for devs we need to understand at least basics to promote it....

Also I think DCDAO members should have some sort of insight in what is the future of Darwinia. Do we plan to sell tech in other ecosystems( Avalanche...) or do we primarily focus on Dotsama or sth else. Like Maduksjay said we need to know a path route to guide people on the road.

Also where can we check how much does DAO have in Treasury?

I like the idea of Town Hall,but can't imagine howit would work. Could you please provide more details?

**Reply by hackfisher** [reply]
> Also where can we check how much does DAO have in Treasury?

You can get to know these information from this page on Snapshot (Found some issue with $RING price feed integration with snapshot @furoxr ): 

https://snapshot.org/#/darwiniacommunitydao.eth/treasury

https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x045f364dcbc7c1d41d51d78fe85473e4a5aa49018aecc2b3bee0d97496d04921

https://app.dework.xyz/darwinia-dao/finance-2561/view/board-le1277q0

**Reply by SasoLithops** [reply]
Thank you for links.

Is it possible we only have 1700$ in Polygon treasury? Does this ammount from treasury only go for paying managers and ambassadors every month? If that is so how will we ever have any Ring/Kton in treasury?
I am sorry I probably missed this information but where does DCDAO get funds from? Itering?

Can we get some funds from Darwinia treasury to become more self efficient?

And hopefully new tokenomics would also reward some funds to DCDAO until it can stand on its own.

**Reply by hackfisher** [reply]
Yes, currently most of DCDAO are funded from donations from Itering and Ecosystem Growth Multisig, and for sure DCDAO could and should apply funding from network's on-chain Treasury for a more sustainable way.

**Reply by SasoLithops** [reply]
Thank you team for funding it. I think that application for grant is a first to start moving things on. 
Team members are part of DCDAO right? Do you plan to stay involved or let DCDAO run by itself. 
My personal opinion is it would be great if we were in it together team and community. 

Can we perhaps count on some help from our devs? We will be able to pay them from DCDAO treasury for their contribution right? Via bounties for certain project DCDAO will need done...

**Reply by hackfisher** [reply]
My understanding is that only the community members who applied and was granted the NFT Profile tokens are DCDAO members and able to participate the DAO governance, there might be overlap between different teams/orgs, but probably are not the same and changing from times to time.

The proposal itself could hire someone/company to support it, however, it is highly correlated with the availability of candidates, and solutions/resources/delivery should be addressed by the proposal/proposer.

**Reply by SasoLithops** [reply]
But isn't point of decentralization that anyone can join DCDAO? Why do we need to have an entity that decides who can join and who can't?
Or do you keep this as a safety for some whale joining DCDAO and doing ill minded proposal that could only benefit him/her?

**Reply by hackfisher** [reply]
> But isn't point of decentralization that anyone can join DCDAO? Why do we need to have an entity that decides who can join and who can't?
> Or do you keep this as a safety for some whale joining DCDAO and doing ill minded proposal that could only benefit him/her? 

I think the government/decentralized rules can be changed by the DAO itself after founding SBT token holders. DCDAO choose this way based on the ideas of soul bounded tokens, I think the member join proposal from @furoxr is only an technical means to verify soul bounded profiles like an oracle considering there is no better way, It should work as objective as much as possible, any contributor recognized can have the SBT token, same with other DAO limiting specific holders of SOME token, should not be anyone which can cause paralysis and unavailability.

**Reply by SasoLithops** [reply]
Understood. Thank you for your time, answers and transparency.

**Reply by Laki389** [reply]
> But isn't point of decentralization that anyone can join DCDAO? Why do we need to have an entity that decides who can join and who can't? Or do you keep this as a safety for some whale joining DCDAO and doing ill minded proposal that could only benefit him/her?

Check out this discussion. https://github.com/orgs/dcdao/discussions/22 A bunch of accounts that I'm seeing for the first time, with similar comments "Interesting proposal", "Very interesting", "Very good proposal!", now let's assume that everyone has the right to vote or propose a vote on Snapshot, one person with thousands of ethereum addresses can propose a vote by himself and outvote everyone else. @SasoLithops

**Reply by SasoLithops** [reply]
Usually these people (bots) are not whales. They tend to check if they can do ill minded proposals, but usually there are more of us who care about Darwinia and would outvote them I am sure about that. I have 7.5 M power, you too Jesse, Dustin even more... I know I would lock my Ring for another 3 years to outvote them,not sure they would do the same. 
But I agree start is the toughest and some security measures are good to be set.

**Reply by Laki389** [reply]
> Usually these people (bots) are not whales. They tend to check if they can do ill minded proposals, but usually there are more of us who care about Darwinia and would outvote them I am sure about that. I have 7.5 M power, you too Jesse, Dustin even more... I know I would lock my Ring for another 3 years to outvote them,not sure they would do the same. 

Voting as it is now is using those DCDAO NFTs we  got. I have 1 vote, you have 1, Jesse, Cindy, all the developers, they all have 1 vote. So that according to the current rules, if everyone who applied is not checked and all are approved, it will happen to us that someone who has nothing with Darwinia, who is hearing about it for the first time, will come with thousands of addresses and vote for anything.

**Reply by SasoLithops** [reply]
True if we keep current voting system.

**Reply by samestshares** [reply]
That is a very valid answer. Can we say everything about Darwinia is all about Developers? How about those that have no understanding of programming but cherish to know more about Darwinia and contribute, is there any program to assist this kind of people since we operate a decentralized system and governance everyone should be welcome.

**Reply by SasoLithops** [reply]
@Cindy884 ☝️

**Comment by Laki389** [comment]
> Is it possible we only have 1700$ in Polygon treasury? 

Hey Saso, as far as I could notice all 3 addresses (on ethereum, arbitrum and polygon) also have $Ring. That 1700 $ you see is just those 1700 usdt on Polygon.  @SasoLithops

**Reply by SasoLithops** [reply]
Oh ok. Could you please let me(us) know,what are the ammounts we have? We could maybe do a proposal to fund some good artist to do an NFT that we could issue(still don't know where maybe where Evolution items are or if Astar building something big) but could bring some more ammount of $ value into treasury. This NFT should have some utility. I thought maybe we could collect points from members activity and let's say top 30 could be in a monthly lottery(just one simple idea) I also think we should bond some of the Ring we have (get) to earn Kton.  Also I think like @hackfisher mentioned we should apply for grant for DCDAO from Darwinia treasury.That is a first. What do you all think?

**Reply by hackfisher** [reply]
Posting a draft of the proposal on dcdao/discussion to test the water before proposal on snapshot is always a good practice.

Other voters can give feedback to or help improve it, the fiscal assistant members also can also give advice on budget and funding.

**Reply by SasoLithops** [reply]
I agree. Awesome. Time to try 😉

**Reply by Cindy884** [reply]
good idea, welcome you can create a post to let us know more details!

**Reply by SasoLithops** [reply]
You mean idea about NFT or something in that direction?

**Comment by JuneCribb** [comment]
I love the idea of co-learning and co-creating, DCDAO will help a lot of us community members who aren't devs to understand better. 

I can also suggest Trivia as a form of learning for the community. ( of course, members have to vote.) 
It will be an interactive and fun section while learning more. 

For the community town hall, webinar or google meet can meet up the requirement.

**Comment by Nisogok** [comment]
Modern DAO is about freedom, decentralization and the choice of each of us. Management and education should be such that each participant is motivated to develop and contribute to web3.0 DAO. 
The idea of cooperative learning is a very good solution!

**Comment by samestshares** [comment]
Beautiful submission and a comprehensive piece.
I saw in the discussion, a school curriculum in which anyone can become a teacher provided he/she has a clear understanding of Web3, Darwinia and DCDAO, It's sure there are many that have a clear account of Web3 (non-technical, I mean not a developer), and in one way wanted to use their ideas, skills (content creation or design graphic) to promote the activities with the Darwinia community or DCDAO, how can this kind of people be home in the Darwinia to be able to understand every technicality around Darwinia ecosystem?

**I am curious about this!**

**Reply by JuneCribb** [reply]
That's the reason for DCDAO. 

 Everyone gets to learn about Darwinia and as said in the discussions, community calls will be held. 

Wanting to promote Darwinia should be a personal contribution in my opinion, e.g. tweeting about Darwinia or the proposed campaign.

**Reply by samestshares** [reply]
When you say sharing Darwinia through Twitter or any social media is personal contribution, then you are saying Darwinia don't need online presence in terms of community member promoting it except on personal dealing, if it's personal then one can decide not to share Darwinia on social which is one of the key to increase adoption, then how do we except more people to know hear of Darwinia if we did not see it as community work, I think is not cool. Online presence in terms of social media is a community service as far as I am concern is not a personal dealings.

**Reply by samestshares** [reply]
That's epic buddy! Let's do this together
